vocab/vocab.csv

```diff
@@ -1379,6 +1379,8 @@ GP05MOAS-PG576-04-FLORTO000,Global Station Papa,Mobile Assets,Profiling Glider 5
 GP05MOAS-PG576-05-NUTNRM000,Global Station Papa,Mobile Assets,Profiling Glider 576,Nitrate,Satlantic,SUNA V2,0,200
 GP05MOAS-PG576-06-PARADM000,Global Station Papa,Mobile Assets,Profiling Glider 576,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
 GS01SUMO,Global Southern Ocean,Apex Surface Mooring,,,,,0,4800
+GS01SUMO-RIC21,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,,,,12,12
+GS01SUMO-RIC21-00-CPMENG000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Platform Controller,WHOI,Communications and Power Manager,12,12
```

Comment:
This looks right, but I think these 2 lines should be inserted before GS01SUMO-RID27-00-DCLENG000 below.